h2o-genmodel/src/main/java/hex/genmodel/algos/glm/GlmMojoModel.java

```diff
@@ -57,10 +57,11 @@ void init() {
     for(int i = _cats; i < _beta.length - 1 - noff; ++i)
       eta += _beta[noff + i] * data[i];
     eta += _beta[_beta.length - 1]; // reduce intercept
+    eta += offset;
 
     double mu = _linkFn.eval(eta);
 
-    if (_binomial || _family.equals("fractionalbinomial")) {
+    if (_binomial || "fractionalbinomial".equals(_family) || "quasibinomial".equals(_family)) {
```

Comment:
I think this comparison should be moved to line 23:

_binomial = _family.equals("binomial");

we do not want to do expensive comparisons for each scored row

Reply:
Good point!